### ISSUE TITLE
Improve documentation in general

### ISSUE THREAD

**Issue by maria-le** [firebase/polymerfire#85]
I'm finding it very difficult to use the `firebase-query` element because the documentation is very thin. Most of the properties have little to no documentation at all.

**Comment by hanoncs** [comment]
I completely agree. We need it much more detailed demos of how to use the different firebase elements together.

**Comment by Mygod** [comment]
Also `note-app` demo is too complicated for a demo imo.

**Comment by garcianavalon** [comment]
+1! The auth element is pretty complex and documentation doesn't cover barely anything but element creation. Can I help somehow?

**Comment by esd100** [comment]
Could you guys upgrade the documentation for `firebase-query`? I'm finding it very difficult to use.

**Comment by esd100** [comment]
Could you guys upgrade the documentation for firebase-document too.

**Comment by carlardron** [comment]
Yep. Could not get started with the module.
